### ISSUE TITLE
Ores only drop Precious Stones 

### ISSUE THREAD

**Issue by idkartist3D** [marcelo-mason/PreciousStones#833]
I have searched and searched and have no idea how to disable it, but whenever, for example, a player mines a coal ore, even when they don't have a silk touch pick, it just gives them a sky cannon. I don't know if this can be turned on or off, but having to smelt coal ore to get coal is getting a bit tiring.

The coal ore drops sky cannons, the iron ore drops the launch pad, the lapis ore drops air of atlantis, and for some reason when you mine nether quartz ore it gives you the nether quartz ore block but it calls it a city plot. Oddly enough, redstone ore has regular drops. No idea why that is.

Basically, is there a way to enable ores only dropping precious stones with a silk touch? I remember playing on a server a while ago that did this but I have no idea how...

**Comment by marcelo-mason** [comment]
Remove the *[meta-autoset](http://preciousstones.shoutwiki.com/wiki/Field_Flag_Reference#meta-autoset)* flag from those fields.
